packet/flatcar-linux/kubernetes/workers/workers.tf

```diff
@@ -10,8 +10,8 @@ resource "packet_device" "nodes" {
   always_pxe       = "false"
   user_data        = "${data.ct_config.install-ignitions.rendered}"
 
-  # If not present in the map, it uses "" that means no reservation id
-  hardware_reservation_id = "${lookup(var.reservation_ids, format("worker-%v", count.index), "")}"
+  # If not present in the map, it uses ${var.reservation_ids_default}
```

Comment:
Missing period at end of sentence.

Reply:
Fixed it, but I'd vote to relax this (part of another discussion). I don't think it is worthwhile